default.json

```diff
@@ -95,7 +95,7 @@
     {
       "fileMatch": ["\\.?aqua\\.ya?ml"],
       "matchStrings": [
-        " +['\"]?(version|ref)['\"]? *: +['\"]?(?<currentValue>[^'\" \\n]+?)['\"]? +# renovate: depName=(?<depName>[^\\n]+)",
```

Comment:
`ref` is used to specify the registry version.
`go` datasource shouldn't be used to update registries.